src/Question.svelte

```diff
@@ -35,7 +35,6 @@
   <div
     class="h-full justify-center items-center question-grid min-w-7xl pt-8 mx-auto bg-app-teal-500"
   >
```

Comment:
then add a div outside here something like:

```
<div class="h-36">
```

the width should be the width of the other component. so there needs to be some adjustment to the html. 
the table and the sun component should have the same width
use something like `min-w-7xl` or similar so make it a bit bigger.